src/mainwindow.cpp

```diff
@@ -244,7 +245,7 @@ void MainWindow::start_web_server() {
     // Start web server
     thread.core = core;
     thread.start();
-    sleep (1);
```

Comment:
You can also use r_sys_sleep, which us portable way using r2 api but i think its better to stick with the qt apis